Resources/config/yui.yml

```diff
@@ -1078,3 +1078,36 @@ system:
                 ez-app-extension-admin:
                     requires: ['ez-serversideview', 'ez-serversideviewservice', 'ez-app-extension']
                     path: %ez_platformui.public_dir%/js/apps/extensions/ez-app-extension-admin.js
+                ez-userprofileview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'event'
+                        - 'event-tap'
+                        - 'ez-usermenuview'
+                        - 'userprofileview-ez-template'
+                    path: %ez_platformui.public_dir%/js/views/ez-userprofileview.js
+                ez-usermenuview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'event'
+                        - 'event-tap'
+                        - 'event-outside'
+                        - 'template-micro'
+                        - 'ez-usermenuitemview'
+                        - 'usermenuview-ez-template'
+                    path: %ez_platformui.public_dir%/js/views/ez-usermenuview.js
+                ez-usermenuitemview:
+                    requires:
+                        - 'ez-templatebasedview'
+                        - 'usermenuitemview-ez-template'
+                    path: %ez_platformui.public_dir%/js/views/ez-usermenuitemview.js
+                userprofileview-ez-template:
```

Comment:
For better code organization, I would put templates just after the module it's used by.